src/marqo/core/unstructured_vespa_index/unstructured_vespa_index.py

```diff
@@ -234,6 +235,12 @@ def _get_score_modifiers(marqo_query: MarqoQuery) -> \
 
     def _to_vespa_lexical_query(self, marqo_query: MarqoLexicalQuery) -> Dict[str, Any]:
         def _get_lexical_search_term(marqo_query: MarqoLexicalQuery) -> str:
+
+            if not marqo_query.or_phrases and not marqo_query.and_phrases:
+                return 'false'
+            if marqo_query.or_phrases == ["*"] or marqo_query.and_phrases == ["*"]:
```

Comment:
This will trigger on edge cases like:
`q='"exact" *'` or `q='"*" optional'`. Specifically, we only want the "fetch all" behavior to happen on `q='*'`. You may have to test this, but I think this means:
```
marqo_query.or_phrases == ["*"] and not marqo_query.and_phrases
```

Reply:
yes you're right! That was my initial implementation as well, but changed it to the above. Will revert. Thanks Joshua